apps/funding-platform/src/index.ts

```diff
@@ -0,0 +1,85 @@
+import { AccessTokenService } from "./access-token";
+import { getWallet } from "./blockchain";
+import { DBInstance } from "./db";
+import * as api from "./entry-server";
+import { checkFreshRequests } from "./queue";
+import { RequestService } from "./request";
+import { utils } from "rpch-common";
+
+const { log } = utils.createLogger(["funding-platform", "index"]);
+const { SECRET_KEY, WALLET_PRIV_KEY } = process.env;
+const port = 3000;
+const MAX_AMOUNT_OF_TOKENS = 100;
+const TIMEOUT = 30;
+let running = false;
+
+const handleRunning = (state: boolean) => {
+  running = state;
+};
+
+export const start = async (ops: {
+  _entryServer: {
+    entryServer: typeof api.entryServer;
+  };
+  _AccessTokenService: typeof AccessTokenService;
+  _RequestService: typeof RequestService;
+  _db: DBInstance;
+  _secretKey: string;
+  _privateKey: string;
+}) => {
+  // init services
+  const accessTokenService = new ops._AccessTokenService(
+    ops._db,
+    ops._secretKey
+  );
+  const requestService = new ops._RequestService(ops._db);
+  const wallet = getWallet(ops._privateKey);
+  // init API server
+  const app = api.entryServer({
+    accessTokenService,
+    requestService,
+    walletAddress: wallet.address,
+    maxAmountOfTokens: MAX_AMOUNT_OF_TOKENS,
+    timeout: TIMEOUT,
+  });
+  setInterval(() => {
+    if (!running) {
+      checkFreshRequests({
+        requestService: requestService,
+        signer: wallet,
+        confirmations: 1,
+        changeState: handleRunning,
+      });
+    }
+  }, 60e3);
+  app.listen(port, () => {
+    log("entry server is up");
+  });
+};
+
+const main = () => {
+  if (!SECRET_KEY) {
+    throw Error("env variable 'SECRET_KEY' not found");
+  }
+  if (!WALLET_PRIV_KEY) {
+    throw Error("env variable 'WALLET_PRIV_KEY' not found");
+  }
+  // init db
+  const dbInstance = {
+    data: {
+      requests: [],
+      accessTokens: [],
+    },
+  } as unknown as DBInstance;
+
+  start({
+    _entryServer: api,
```

Comment:
prefixing with underscore often means "this is private", would rather try to go with a different scheme, maybe something like https://github.com/Rpc-h/RPCh/blob/13681cf63b1c88f8f9c73eaf8c6e9889f0ac4e3f/apps/exit-node/src/index.ts#L12